frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboardPodCount.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Bullseye } from '@patternfly/react-core';
+import { Firehose, FirehoseResult, LoadingInline } from '@console/internal/components/utils';
+import { PodModel } from '@console/internal/models';
+import './MonitoringDashboardPodCount.scss';
+import { PodKind } from '@console/internal/module/k8s';
+
+interface MonitoringDasboardPodCountProps {
+  namespace: string;
+}
+
+interface PodCountProps {
+  pods?: FirehoseResult<PodKind[]>;
+}
+
+export const PodCount: React.FC<PodCountProps> = ({ pods }) => {
```

Comment:
We usually have one component per file, will it make sense here as well?  I know here `MonitoringDasboardPodCount`  we just have `Firehose`. WDYT?

Reply:
In ```MonitoringDasboardPodCount``` just have ```Firehose``` that is why put it in one file.

Reply:
I'm ok with multiple components per file if it's used internally. The idea being that only the default export should ever be consumed elsewhere.